dashboard/test/ui/features/teacher_student_toggle.feature

```diff
@@ -112,7 +112,6 @@ Scenario: Toggle on Lockable Level with new teacher panel
   And element ".level-group" is not visible
 
   Then I am on "http://studio.code.org/s/allthethings"
-  And I select the first section
```

Comment:
strangely, this step ends up navigating from a url like `/s/allthethings?section_id=964056` to `/s/allthethings?section_id=undefined`, because `$('.content select')` is not present on the script overview page. Once this passes DTT, I'll remove this step from more places.